cpp/velox/memory/VeloxMemoryPool.h

```diff
@@ -21,11 +21,14 @@
 #include "velox/common/memory/Memory.h"
 
 namespace gluten {
+class WrappedVeloxMemoryPool;
 
 std::shared_ptr<facebook::velox::memory::MemoryPool> AsWrappedVeloxMemoryPool(
     MemoryAllocator* allocator,
     const facebook::velox::memory::MemoryPool::Options& options);
 
-std::shared_ptr<facebook::velox::memory::MemoryPool> GetDefaultWrappedVeloxMemoryPool();
+std::shared_ptr<WrappedVeloxMemoryPool> GetDefaultWrappedVeloxMemoryPool();
+
+std::shared_ptr<facebook::velox::memory::MemoryPool> GetDefaultLeafWrappedVeloxMemoryPool();
```

Comment:
I don't yet have much context but would you like to explain a bit more about this change? Why do we have to add this function?

Is this because "aggregate pool" is no longer accepted by some APIs in Velox?

Reply:
If we much have two default pools, can we rename the functions using the same convention,

e.g.

```
std::shared_ptr<facebook::velox::memory::MemoryPool> GetDefaultWrappedVeloxAggregateMemoryPool();
std::shared_ptr<facebook::velox::memory::MemoryPool> GetDefaultWrappedVeloxLeafMemoryPool();
```

I put "Aggregate"/"Leaf" towards right since the attribute "Velox" is more important than "Aggregate"/"Leaf" for a "MemoryPool".

Reply:
Yes, you are right.  Here is some context for you. In previous velox version. `GetDefaultWrappedVeloxMemoryPool` will create a memory pool which can allocate memory and add child. But now only aggregate pool can add child. So I think we need create a new function to create a wrapped leaf pool for allocate memory.

GetDefaultWrappedVeloxMemoryPool -> aggregate pool
GetDefaultLeafWrappedVeloxMemoryPool  -> leaf pool

Reply:
> std::shared_ptr<facebook::velox::memory::MemoryPool> GetDefaultWrappedVeloxAggregateMemoryPool();
> std::shared_ptr<facebook::velox::memory::MemoryPool> GetDefaultWrappedVeloxLeafMemoryPool();

I will rename it.